sdk-python/test.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+import livescoreapi
+
+
+def livescores_api():
+    client = livescoreapi.Livescores('http://livescore-api.com/api-client/', '6r94GgdPiJ5ciqdx', '70Qx0KjZN2uD6jfdLohrFuhXem9wNm4U')
```

Comment:
Next time when you push do not commit API key and Secrets as they are supposed to be a secret

Reply:
And in general this file should not be commited.